driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -149,6 +163,16 @@ public ConfigBuilder withMinIdleTimeBeforeConnectionTest( long milliSecond )
         }
 
         /**
+         * Enable TLS in all connections with server
+         * @param value
+         */
+        public ConfigBuilder withTLSEnabled( boolean value )
```

Comment:
We will probably only have two levels: no TLS, TLS trust-on-first-use. Our server also use the same strategy, so does the client.